src/components/viewer/DrawTools.vue

```diff
@@ -783,23 +783,38 @@ export default {
       this.copiedAnnot = feature.properties.annot.clone();
       this.$notify({type: 'success', text: this.$t('notif-success-annotation-copy')});
     },
-    convertLocation(originalLocation, image) {
+    async convertLocation(copiedAnnot, destImage) {
       /* If we want to paste in the same image but in another slice */
-      if (this.image.id === this.copiedAnnot.image) {
-        return this.copiedAnnot.location;
+      if (destImage.id === copiedAnnot.image) {
+        return copiedAnnot.location;
       }
 
-      let geometry = new WKT().readGeometry(originalLocation);
-      let wrapper = this.imageWrapper;
+      /* Compute the rescaling factor if the resolution is known for both images */
+      let scale = 1;
+      let srcImage = await ImageInstance.fetch(copiedAnnot.image);
```

Comment:
It will work but it's a shame to always re-fetch data from server at every paste action, while data is already in the store.

You could save the `index` of the viewer of the source image when the action is copied, and then you can get the image instance without refetching it with `this.viewerWrapper.images[copiedAnnotImageIndex].imageInstance`

Reply:
Changed in cbc18f3

Reply:
Yes it is correct, it will fail if the image is closed. I didn't think about this issue.
Fix in d53d0d5 by storing `copiedAnnotImageInstance` instead of the index.